src/components/SeparatorSchemaEnhancer.js

```diff
@@ -0,0 +1,30 @@
+import { addStyling } from '@plone/volto/helpers/Extensions/withBlockSchemaEnhancer';
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+  noLine: {
+    id: 'Hide Separator Line',
+    defaultMessage: 'Hide Separator Line',
+  },
+  align: {
+    id: 'Alignment',
+    defaultMessage: 'Alignment',
+  },
+});
```

Comment:
@danalvrz Since you added new messages (translatable strings), we need to update the files in the `locales` folder to include the new messages. To do that, run `yarn i18n` at the top level of volto-separator-block